examples/lines_bars_and_markers/stem_plot.py

```diff
@@ -12,7 +12,7 @@
 x = np.linspace(0.1, 2 * np.pi, 41)
 y = np.exp(np.sin(x))
 
-plt.stem(x, y)
+plt.stem(x, y, use_line_collection=True)
```

Comment:
Needed to prevent warning

> /home/circleci/project/examples/lines_bars_and_markers/stem_plot.py:15: UserWarning: In Matplotlib 3.3 individual lines on a stem plot will be added as a LineCollection instead of individual lines. This significantly improves the performance of a stem plot. To remove this warning and switch to the new behaviour, set the "use_line_collection" keyword argument to True.